src/platform/graphics/texture.js

```diff
@@ -773,6 +790,12 @@ class Texture {
         options.face ??= 0;
         options.mode ??= TEXTURELOCK_WRITE;
 
+        if (options.mode === TEXTURELOCK_NONE) {
+            Debug.log("pc.Texture#lock: To unlock a texture, call texture.unlock(). Defaulting to 'TEXTURELOCK_READ'.", this);
+            options.mode = TEXTURELOCK_READ;
+        }
```

Comment:
I'd just have
```
Debug.assert(options.mode === TEXTURELOCK_READ || options.mode === TEXTURELOCK_WRITE);
```
as nothing else is allowed.

Reply:
Done!